packages/html/ds/src/footer/helpers.html

```diff
@@ -0,0 +1,15 @@
+{% macro hasTwoCols(twoCols) %}
+  {% if twoCols %}
+    sm:gi-col-span-2
+  {% endif %}
+{% endmacro %}
+
+{% macro hasTwoColsLinks(twoCols) %}
+  {% if twoCols %}
+    sm:gi-columns-2
+  {% endif %}
+{% endmacro %}
+
+{% macro govieFooterLink(props) %}
+  <a class="gi-underline gi-underline-offset-sm" href="{{ props.href }}">{{ props.label }}</a>
+{% endmacro %}
```

Comment:
nitpick: Same as above for the missing line.

Reply:
Because we have multiple macros in a single file, I thought it would be more readable if we have an empty lines between macros ? Otherwise it gets quite clunky and hard to read